manifest/ManifestParser.ts

```diff
@@ -0,0 +1,62 @@
+import {Manifest, SourceTimeout} from "../types";
+
+export type TokensBySource = { [source: string]: string[] };
+
+export default class ManifestHelper {
+
+  // This function converts tokens from manifest to object with the following
+  // type: { <SourceName>: <Array of tokens to fetch from source> }
+  static groupTokensBySource(manifest: Manifest): TokensBySource {
```

Comment:
I'm wondering if we should also somehow validate the source name (if there is a fetcher registered for a given source), in case someone makes a typo in the name of the source